packages/supi/src/api/storeAdd.ts

```diff
@@ -0,0 +1,59 @@
+import normalizeRegistryUrl = require('normalize-registry-url')
+
+import {streamParser} from '@pnpm/logger'
+import {StoreController} from 'package-store'
+
+import parseWantedDependencies from '../parseWantedDependencies';
+import {ReporterFunction} from '../types'
+
+export default async function (
+  fuzzyDeps: string[],
+  opts: {
+    prefix: string,
+    registry?: string,
+    verifyStoreIntegrity: boolean,
+    reporter?: ReporterFunction,
+    storeController: StoreController,
+  },
+) {
+  const reporter = opts && opts.reporter
+  if (reporter) {
+    streamParser.on('data', reporter)
+  }
+
+  const deps = await parseWantedDependencies(fuzzyDeps, {
+    allowNew: true,
+    currentPrefs: {},
+    defaultTag: '',
```

Comment:
you could pass in the tag config from pnpm. It is a config that is "latest" by default.